src/components/Rating.vue

```diff
@@ -1,11 +1,16 @@
 <template lang="pug">
 	ul.uk-padding-remove.uk-margin-remove.uk-inline-block(uk-tooltip, :title="overall | rating")
 		li(v-for="n in stars").uk-inline-block
-			span(:class="(n <= overall) ? '-on' : '-off'", uk-icon="icon: star; ratio: 0.8").star
+			span(:class=getIconClass(n), uk-icon="icon: star; ratio: 0.8").star
 </template>
 
 <script>
 	export default {
+		methods: {
+			getIconClass(n) {
+				return (this.n <= this.overall) ? '-on' : '-off';
```

Comment:
```suggestion
				return (n <= this.overall) ? '-on' : '-off';
```
you need to reference the provided param, not the data prop anymore.